src/descrambler/capmt.c

```diff
@@ -777,14 +787,33 @@ capmt_service_destroy(th_descrambler_t *td)
 }
 
 static void
+capmt_send_client_info(capmt_t *capmt)
+{
+  char info_version[255];
+  snprintf(info_version, sizeof(info_version), "Tvheadend %s", tvheadend_version);
```

Comment:
Just cometic - one char[] should be enough. Just use buf + 7 and sizeof(buf) - 7 in this snprintf and put everything to this buffer:

*(uint32_t *)(buf + 0) = htonl(DVBAPI_CLIENT_INFO);
*(uint16_t *)(buf + 4) = htons(DVBAPI_PROTOCOL_VERSION);

The accesses seem aligned here..